src/main/java/seedu/address/model/item/Item.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.item;
+
+import java.util.Objects;
+
+import seedu.address.model.tag.UniqueTagList;
+
+/**
+ * Represents a Item in the address book. Guarantees: details are present and
+ * not null, field values are validated.
+ */
+public abstract class Item {
+
+
+    private UniqueTagList tags;
```

Comment:
Should we also put a boolean `isComplete` here? (And maybe in future updates the `priorityLevel`)?